app/src/main/java/ru/krivocraft/kbmp/fragments/LargePlayerFragment.java

```diff
@@ -386,19 +387,19 @@ private void refreshUI() {
         compositionNameTextView.setSelected(true);
         compositionAuthorTextView.setText(getTrackArtist());
 
-        int duration = TimeUtils.getSeconds(getTrackDuration());
+        int duration = new Milliseconds(getTrackDuration()).seconds();
         compositionProgressBar.setOnSeekBarChangeListener(this);
         compositionProgressBar.setMax(duration);
 
         updateStateShowers();
     }
 
     private void updateStateShowers() {
-        int progress = TimeUtils.getSeconds(trackProgress);
-        int duration = TimeUtils.getSeconds(getTrackDuration());
+        int progress = new Milliseconds(trackProgress).seconds();
+        int duration = new Milliseconds(getTrackDuration()).seconds();
 
-        compositionProgressTextView.setText(TimeUtils.getFormattedTime(progress));
-        compositionDurationTextView.setText(TimeUtils.getFormattedTime((duration - progress) / 1000));
+        compositionProgressTextView.setText(new Seconds(progress).formatted());
+        compositionDurationTextView.setText(new Seconds((duration - progress) / 1000).formatted());
```

Comment:
Do we need this conversion /1000 here?

Reply:
Looks pretty much like an error.  
If `progress` is measured in seconds, then the second line looks suspicious. If in millis - the first one is erroneous and `Seconds` class is coded incorrectly. 

There should be a separate isssues for this. @zelenyhleb, please bake one.